roles/forklift/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
+forklift_directory: "{{ role_path | dirname | dirname }}"
```

Comment:
I now realized this might break use cases where forklift is used as a library (sat-deploy)

Reply:
You can test via https://github.com/pulp/pulplift

Reply:
Why the double dirname?

Reply:
Because it's in `roles/forklift`